<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtDLGExceptionAskClass</class>
 <widget class="QDialog" name="qtDLGExceptionAskClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>[ Nanomite ] - Unhandled Exception Found!</string>
  </property>
  <widget class="QPushButton" name="pbIgnore">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ignore</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBreak">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Break</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbApp">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Send to App</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbAlways">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Remember decision for this Exception</string>
   </property>
  </widget>
  <widget class="QLabel" name="lableState">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>440</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qtDLGNanomite.qrc"/>
 </resources>
 <connections/>
</ui>
